docs/src/filters/token_router.md

```diff
@@ -82,10 +82,12 @@ For example, a configuration would look like:
 version: v1alpha1
 static:
   filters:
-    - name: quilkin.extensions.filters.capture_bytes.v1alpha1.CaptureBytes # Capture and remove the authentication token
+    - name: quilkin.extensions.filters.capture.v1alpha1.Capture # Capture and remove the authentication token
       config:
-          size: 3
-          remove: true
+        strategy:
+            kind: SUFFIX
+            size: 3
+            remove: true
```

Comment:
is it possible to use the [proposed schema here](https://github.com/googleforgames/quilkin/pull/387#issuecomment-910179544) that groups the fields by strategy? this look like we'll have 1 flat object where all fields are together and we'll have to document which belongs to which strategy